src/plugins/jsx/index.js

```diff
@@ -342,10 +356,12 @@ pp.jsxParseElementAt = function(startPos, startLoc) {
   let closingElement = null;
 
   if (!openingElement.selfClosing) {
-    contents: for (;;) {
+    contents:
```

Comment:
This will be fixed in the next release: https://github.com/prettier/prettier/pull/860